### PR TITLE
Update AdMob sample for 8.0

### PR DESCRIPTION
**Pull request by morganchen12**
Googlers see b/186862697 for details.

### CODE REVIEW

**Review by paulb777 — COMMENTED** [review]
From the travis failure, it looks like a missing linkage for StoreKit?  Perhaps a problem with the Xcode project's settings blocking the frameworks from the pod specifications?

File: admob/AdMobExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -135,6 +140,8 @@
 				10360400224D9A1600E6532E /* AdMobExampleUITests */,
 				1036040E224D9A2900E6532E /* AdMobExampleSwiftUITests */,
 				1036038D224D95D300E6532E /* Products */,
+				583D576FC6131BC336AF9238 /* Pods */,
```
Comment (paulb777) [inline admob/AdMobExample.xcodeproj/project.pbxproj:143]: Delete

File: admob/AdMobExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -210,6 +217,21 @@
 			path = AdMobExampleSwiftUITests;
 			sourceTree = "<group>";
 		};
+		583D576FC6131BC336AF9238 /* Pods */ = {
+			isa = PBXGroup;
+			children = (
+			);
+			path = Pods;
+			sourceTree = "<group>";
+		};
```
Comment (paulb777) [inline admob/AdMobExample.xcodeproj/project.pbxproj:226]: Delete

**Review by paulb777 — COMMENTED** [review]
Two ways to deal with the lingering `Pods` proactively. Delete the red Pods in the Xcode UI after `pod deintegrate`. Or close Xcode before running `pod deintegrate`

File: admob/AdMobExample.xcodeproj/project.pbxproj (hunk 2 above)
Comment (morganchen12) [reply admob/AdMobExample.xcodeproj/project.pbxproj]: Done.

File: admob/AdMobExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (morganchen12) [reply admob/AdMobExample.xcodeproj/project.pbxproj]: Done.

**Comment by paulb777** [comment]
Do you know why merging master fixed the previous CI issues?

**Comment by morganchen12** [comment]
The previous CI issues ran on travis and merging master moved them to GitHub Actions.

That's about all I know.

**Review by paulb777 — APPROVED** [review]

**Pull request merged**